test/e2e/create_test.go

```diff
@@ -340,6 +350,16 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, expectError, checkVe
 				return false, nil
 			}
 
+			// verify the node CSR associated with the node is not waiting for approval.
+			csr, err := tc.findNodeCSR(node.GetName())
+			if err != nil {
+				log.Printf("error finding node CSR: %s", err)
+				return false, nil
+			}
+			if tc.isPending(csr) {
+				log.Printf("node csr is waiting for approval")
+				return false, nil
+			}
```

Comment:
Please add comment with the Jira story link to clearly cal out that this section needs to be removed. This applies to all the sections that need to be removed. I am asking for this as I suspect that a clean revert won't work if there is more code churn in this file.

Reply:
I was thinking this was a check that would be useful for the CSR approval ticket. I will add a TODO suggesting its removal

Reply:
That's a good point. However that decision should be made by whoever is working on the CSR story.